cookiecutter.json

```diff
@@ -1,8 +1,8 @@
 {
-  "full_name": "UCL ARC",
-  "email": "",
-  "github_username": "{{ cookiecutter.full_name.replace(' ', '-') }}",
-  "project_name": "Python Template",
+  "author_name": "",
+  "author_email": "",
+  "github_username": "{{ cookiecutter.author_name.replace(' ', '-') }}",
+  "project_name": "",
```

Comment:
ah, CI is failing because there's a test to check that the folder `python-template` exists. Maybe keep the default as "Python template"? Or change the test if you prefer